### ISSUE TITLE
Unable to use with bookwalker

### ISSUE THREAD

**Issue by xxtensazenxx** [MinoMino/mindl#16]
Sorry to open this if the program is no longer being actively developed, but I cannot get it to work with bookwalker.

this is on Windows 10 64bit

I cd into the directory with the program ofc, then run

`mindl -d -z -o username=redacted@email.com -o password=password https://global.bookwalker.jp/ded443a3d3-9fdb-4902-b7bd-69c34967cb5b/`

I get the following output 

```
(20:37:00 ERRO) Found no handler for: https://global.bookwalker.jp/ded443a3d3-9fdb-4902-b7bd-69c34967cb5b/
(20:37:00 FATA) No plugins to select from.
```

Am I missing anything in the arguments or command? or is this project not currently working anymore?

Thanks

**Comment by MinoMino** [comment]
Yep, unfortunately this repo is not being maintained, so if anything I'd be surprised if any of the previously supported sites still work. That being said, I do plan on picking it up again and hopefully get all the sites working again at some point, I'm just not sure when exactly. Hopefully in the near future though.
